src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -286,6 +292,7 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
                 onPromote={this.handleActionPromote}
                 onRestore={this.handleActionRestore}
                 parentName={parentName}
+                onUpgradeClick={onUpgradeClick}
```

Comment:
Minor, but can we alphabetize this prop to stay consistent with the others?

Reply:
yup yup!